Userland/Applications/Calendar/main.cpp

```diff
@@ -112,6 +118,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(toolbar->try_add_separator());
     (void)TRY(toolbar->try_add_action(view_month_action));
     (void)TRY(toolbar->try_add_action(view_year_action));
+    (void)TRY(toolbar->try_add_action(open_settings_action));
```

Comment:
Same here, it doesn't make sense for this to be `void`-ed, I don't know why the previous lines do it that way :confused: